src/ontology/config/prefixes.csv

```diff
@@ -112,7 +112,6 @@ FOODON,http://purl.obolibrary.org/obo/FOODON_
 REX,http://purl.obolibrary.org/obo/REX_
 FOVT,http://purl.obolibrary.org/obo/FOVT_
 KISAO,http://purl.obolibrary.org/obo/KISAO_
-FBcv,http://purl.obolibrary.org/obo/FBcv_
```

Comment:
## The changes
There are 3 such cases of duplicates: Same prefix and URI prefix. From what I understand, removing these should have no effective change on outputs or runtime. Should just result in a codestyle fix.

Reply:
Ok, duplicate!